### ISSUE TITLE
User is not able to modify personal record - it doesn't change after clicking "Зберегти" button

### ISSUE THREAD

**Issue by MPushkar** [ifnul/ums-admission-frontend#88]
Pre-conditions:
- Any personal <b>Record1</b> should be present in system.

Steps to reproduce:
1. Login to System under Admin and go to a "Персони" tab.
2. Click ![lnu_modify_button](https://cloud.githubusercontent.com/assets/12950571/8501140/57b5f09e-21ab-11e5-8c39-5d268842db28.PNG) button for <b>Record1</b>.
3. Modify any field, e.g. "По-батькові" and click "Зберегти" button.
4. Go to "Персони" screen and find <b>Record1</b>.

Actual result:
Value in "По-батькові" is not updated.

Expected result:
Value in "По-батькові" should be updated.

**Comment by ITsvetkoFF** [comment]
Да, система сира нажаль. перевірив. запис помінявся. був Іван Іванович Іваненко, став Іванко Іванович Іваненко. 
Біда відбувається з персоною для якої не внесено адресу. Якщо адресу внесено, то все працює ніби.

**Comment by mkozhukharenko** [comment]
спостерігав що з консолі постійно летять помилки які стосуються адреси, типу

```
Cannot read property 'addressTypeId' of undefined
```

я бачив що директиву яка відповідає за додавання адрес змінювали, з якою метою? Тобто чи пофіксили її ? Чи просто помилки летять через те що адреса не додана?

**Comment by illay1994** [comment]
програма має працювати навіть якщо у персони не введено адресу цілісність даних при цьому залишається

**Comment by mkozhukharenko** [comment]
@illay1994 ти маєш на увазі якщо 1) адреси взагалі немає чи 2) якщо адреса є, але при її виборі юзер не дійшов "до дна" (наприклад зупинився на місті, але дале ще були райони) ?
